src/table/TableBodyWrapper.jsx

```diff
@@ -45,7 +48,7 @@ const TableBodyWrapper = ({
 }) => {
   const { rows, columns } = tableData;
   const hoverEffect = layout.components?.[0]?.content?.hoverEffect;
-  const bodyStyle = useMemo(() => getBodyStyle(layout, theme), [layout, theme.name()]);
+  const bodyStyle = useMemo(() => getBodyStyle(layout, theme, rootElement), [layout, theme.name()]);
   const classes = useStyles(bodyStyle);
```

Comment:
I'd say the `rootElement` is not part of `bodyStyle`. concat it when calling `useStyles` here instead.
`useStyles({ ...bodyStyles, rootElement.height })`

Reply:
sure